share/spice/yoda_speak/yoda_speak.js

```diff
@@ -0,0 +1,29 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_yoda_speak = function(api_result) {
+
+        if (!api_result || api_result.yodish === '' || api_result.error ) {
+            return Spice.failed('yoda_speak');
+        }
+
+        Spice.add({
+            id: "yoda_speak",
+            name: "Answer",
+            data: {
+                title: api_result.yodish,
+            },
+            meta: {
+                sourceName: "Yodish.com",
+                sourceUrl: "http://www.yodajeff.com/pages/talk/yodish.shtml" 
+            },
+           
+            templates: {
+                group: 'text',
+                options: {
+                    content: Spice.yoda_speak.yoda_speak,
```

Comment:
Meaning, use the standard title + subtitle approach. The title should show the answer, the subtitle should indicate what's going on: `Translation to Yoda Speak: <input>`